Lesson Plans/Private/XC.9 Emergency Operations.md

```diff
@@ -23,42 +23,107 @@ Develop knowledge of all abnormal and emergency procedures for the training airc
 > Can be completed at the same time as [[XC.8 Emergency Flight by Instruments - Four Fundamentals]] or [[XC.10 Lost Procedures and Diversion to Alternates]].
 
 ### Lesson Elements
+##### Handling Emergencies
 1. [[FAR 91.3 PIC Responsibility and Authority]]
-3. What is an emergency? When should you declare one?
+2. What is an emergency? When should you declare one?
 	1. When the outcome of the flight is in doubt
-4. Instrument failures (vacuum or pitot-static system)
-5. Fire
-	1. Engine fire
-		1. In air - Emergency Descent (this is the rapid descent procedure)
-			1. Start descent (bank 30°-45°; airspeed >=100 kts; should have positive load factor)
-			2. Then once fire out, execute forced landing (see [[SOLO.15 Emergency Landings]])
-		2. During start on ground
-	2. Electrical Fire
-	3. Wing Tip Fire
-6. Failure
-	1. Engine
-		1. During flight: see [[SOLO.15 Emergency Landings]]
-		2. During takeoff roll
-		3. Immediately after takeoff: Land straight ahead!^[[Cub Theatre: The-Not-That-Impossible Turn](https://www.youtube.com/watch?v=byBh1Qd6IgA) from [[Paul Bertorelli]] of [[AVweb]]]
-	2. Electrical
-		1. Malfunctions or complete failure
-		2. Nordo (No Radio)
-	3. Minimum fuel
-	4. Flap failure
-	5. Retractable gear malfunction
-	6. Door opening in flight
-	7. Low oil pressure
-7. Unsafe Conditions
-	1. [[VFR]] into [[IMC]]
-	2. Icing
-		1. [[How and Why Carburetor Ice Forms]]
-8. Emergency equipment and survival gear
-9. Emergency squawk codes: [[Squawk 7700]], [[Squawk 7600]], [[Squawk 7500]]
+	2. When you need higher priority handling from ATC
+
+##### Something is Wrong!
```

Comment:
I always tell students one big thing is to practice these procedures so they're smooth, but also to try and work through what you'll do if you encounter them in various places. Classic is "Where will you go if you're engine fails now?", but I always tell this story...

I knew a pilot who had an electrical failure over Byron. That pilot had never thought about what to do, so did not have TRACON phone number in his cell phone and had no clue but to treat it like a forced landing right at the non-towered airport. In practice, he should have known he could enter the Mode C veil with permission and he could have called TRACON on his cellphone. It wasn't the wrong choice to land and spend the next 12 hours at Byron, but if he had premeditated this sort of scenario, he might have come up with different solutions that were just as safe.

Reply:
That's a really good reminder.  I wouldn't have thought to do that either.  What is that number? :-)